lib/tasks/one_offs.rake

```diff
@@ -2,5 +2,11 @@
 namespace :one_offs do
   desc "runs all one_offs, remove things from here after they are deployed"
   task run_all: :environment do
+    ChangeReport.all.each do |change_report|
+      change_report.navigator.update!(
```

Comment:
I think you all might have seen this alert, but we just got a release phase failure on the PR app because navigator is nil. may want to do a guard here! https://dashboard.heroku.com/apps/colorado-benefits-stagi-pr-129/activity/releases/12